docker-compose.yml

```diff
@@ -7,11 +7,10 @@ services:
     container_name: Grasa-Django-App
     network_mode: host
     ports:
-      - 80:80
+      - 8000:8000
       - 443:443
```

Comment:
This mixes a standard port with a custom port. I suggest dropping `443:443` since we are only serving the application via port `8000`. We can drop `443` since it is not needed anymore.

Reply:
443 should be opened at a later time so that we can handle HTTPS traffic, and 8000 is being used because our test servers, ie my personal servers, have other services running on port 80.  8000 is the Django default port.

Reply:
In our case, Django serves the application out of one port. We handle HTTP/HTTPS at the web server level in a real deployment environment and can switch between that. From a development POV, Django only binds to a single assigned port.

For example, we could do…

`- 8000:80`

…since this is only used for local development at the moment. HTTPS in development is not required.